### ISSUE TITLE
Feature Request: Please upgrade or provide a "-git" version of sabnzbdvpn

### ISSUE THREAD

**Issue by mathos** [binhex/arch-sabnzbdvpn#2]
Sabnzbd just did a major push about a week ago, it would be nice to have the container updated for that... (Actually if I had my druthers, it would be for a "nzbgetvpn-git" :grin: :sparkles: )

**Comment by binhex** [comment]
The image has been updated, the reason for the delay was because I was waiting on AUR to catch up.
